### PR TITLE
Add clear hover theming to Select and SelectMultiple components

### PR DESCRIPTION
**Pull request by NOMADE55**
## What does this PR do?
Updates the documentation to account for the new theming https://github.com/grommet/grommet/pull/7396

---
Let me know if wording is correct. I based mine on what was already in the docs.

### CODE REVIEW

**Review by MikeKingdom — APPROVED** [review]
That works. Thanks!

File: src/screens/Select.js [hunk 1]
```diff
@@ -644,6 +645,24 @@ const SelectPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="select.clear.hover.background">
```
Comment (britt6612) [inline src/screens/Select.js:648]: this is now `theme.select.clear?.container?.hover` 

https://github.com/grommet/grommet/pull/7397

if we can change the documents to match the PR that got merged :)

File: src/screens/Select.js [hunk 2]
```diff
@@ -644,6 +645,24 @@ const SelectPage = () => (
...
+          <Description>
+            Adjustments to the default clear button background style when the
+            button is hovered.
+          </Description>
+          <GenericBackground />
+        </Property>
+
+        <Property name="select.clear.hover.text">
```
Comment (britt6612) [inline src/screens/Select.js:656]: we can remove the `text`

**Comment by NOMADE55** [comment]
Updated to reflect the merged PR!

**Review by jcfilben — APPROVED** [review]
Looks good!

**Comment by jcfilben** [comment]
Waiting to merge until we are closer to releasing the next version of grommet

**Comment by NOMADE55** [comment]
> Waiting to merge until we are closer to releasing the next version of grommet

Anything missing for me to help on the new version?

**Comment by NOMADE55** [comment]
> Waiting to merge until we are closer to releasing the next version of grommet

Neat, regarding new version, anything I can help with that's missing or none working on?

**Comment by jcfilben** [comment]
> Neat, regarding new version, anything I can help with that's missing or none working on?

Hey @NOMADE55 awesome that you are interested in helping out! I don't think there is anything in particular we need for this next release, but anyway feel welcome if there is an issue you are interested in working on feel free to put up a pull request or post a message on the issue.
At the moment we are working to resolve any accessibility issues in grommet

**Pull request merged**